Project/app/src/main/java/org/ionproject/android/favorites/FavoritesFragment.kt

```diff
@@ -20,4 +36,61 @@ class FavoritesFragment : Fragment() {
         return inflater.inflate(R.layout.fragment_favorites, container, false)
     }
 
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        setupFavoritesListBehaviour(recyclerview_favorites_classes_list)
+        setupCalendarTermSpinner(spinner_favorites_calendar_terms)
+    }
+
+    private fun setupFavoritesListBehaviour(favoritesList: RecyclerView) {
+        //TODO Confirm if this is the right context
+        favoritesList.layoutManager = LinearLayoutManager(context)
+
+        val favoritesListAdapter = FavoritesListAdapter(viewModel)
+        favoritesList.adapter = favoritesListAdapter
+
+        val itemTouchHelper = ItemTouchHelper(
+            FavoritesListAdapter.SwipeToDelete(favoritesListAdapter))
+        itemTouchHelper.attachToRecyclerView(favoritesList)
+
+        var count = 2
```

Comment:
What is this count for? Is it for dealing with some kind of anomaly? Is it a long term solution? If so, please document. If not, leave a TODO to fix this later.